src/util/routeUtils.js

```diff
@@ -9,10 +9,16 @@ import matchPath from 'react-router-dom/matchPath';
 type Resolver<T> = (input: T) => Promise<T>;
 
 // munge a route's 'relative' `path` with the full matchedPath
-const routePath = (route: PlatformRoute, matchedPath: string): string =>
-	`${matchedPath}${route.path || ''}`.replace('//', '/');
-
-export const decodeParams = (params: Object) =>
+const routePath = (
+	route: PlatformRoute,
+	matchedPath: string
+): MatchPathOptions => ({
```

Comment:
although technically `matchPath` can take a string arg, it will convert it into the preferred `MatchPathOptions` object, so I've refactored this function to return a `MatchPathOptions` object directly